src/sonic_ax_impl/mibs/ietf/rfc2863.py

```diff
@@ -98,10 +99,12 @@ def update_data(self):
         Update redis (caches config)
         Pulls the table references for each interface.
         """
-        self.if_counters = {
-            sai_id: Namespace.dbs_get_all(self.db_conn, mibs.COUNTERS_DB, mibs.counter_table(sai_id), blocking=True)
-            for sai_id in self.if_id_map}
-
+        for sai_id_key in self.if_id_map:
+            namespace, sai_id = mibs.split_sai_id_key(sai_id_key)
+            if_idx = get_index_from_str(self.if_id_map[sai_id_key].decode())
+            for db in self.db_conn:
+                if db.namespace == namespace:
```

Comment:
This is not efficient, because you search `self.db_conn` too many times.

Propose 2 alternative solutions:
1. create a map (namespace -> db)
2. use db_index instread of namespace in `get_sai_id_key`

Reply:
Added a map, db_index cannot be obtained directly in this function and will have to  be passed as an additional argument.

Reply:
@qiluo-msft thanks for pointing out. 
we discussed to have map but to avoid new map went with this approach of checking namespace and break considering loop is small.